### PR TITLE
Talon and Tether OSHA Pass

### PR DESCRIPTION
**Pull request by VictorZisthus**
Added numerous air alarms to talon where they were missing, moving some things around to boot.

Added intercoms to mining and cargo so miners can communicate during a power outage.

Added a few security cams to talon.

Hooked up the gas heating/cooling systems to the talon so they actually do something. Moved APC's to compensate and maintain symmetry.

Added Talon to the list of Radio channels that the portable radios on the Talon can communicate with.

Added armguards and leg-guards to the Talon Armory so that the spacesuit isn't the only thing that can cover arms and legs in a firefight.

Added a parts-lathe circuit board to the Engi-Vend on Talon.

Added security cameras to the Gravgen on Tether.

Added 30 sheets of plasteel to Talon's mat storage and a whetstone to it's Armory.

Added requested Blood Vendor and Smartfridge, moving things around to compensate.

Added cardboard and plastic to the service disposal/shipping lanes so crates and boxes can be made.

Added spades to the Hydro lockers. Added plant analyzers for Farmbots.

### CODE REVIEW

File: maps/offmap_vr/talon/talon2.dmm [hunk 1]
```diff
@@ -721,6 +721,24 @@
...
 /turf/simulated/floor/tiled/eris/steel,
 /area/talon/decktwo/central_hallway)
+"bG" = (
+/obj/structure/cable/green{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2"
+	},
+/obj/structure/catwalk,
+/obj/machinery/alarm/talon{
```
Comment (Novacat) [inline maps/offmap_vr/talon/talon2.dmm:731]: ...Why are the tether anomaly lab air alarms being used for talon?

Comment (VictorZisthus) [reply maps/offmap_vr/talon/talon2.dmm:731]: That is a very good question.

**Comment by TheFurryFeline** [comment]
Just making a note that there's been a request for a ChemVator/ChemStorage and Blood Vendor for the Talon.

**Comment by VictorZisthus** [comment]
> 
> 
> Just making a note that there's been a request for a ChemVator/ChemStorage and Blood Vendor for the Talon.

Adding that right now actually!

**Comment by TheFurryFeline** [comment]
> > Just making a note that there's been a request for a ChemVator/ChemStorage and Blood Vendor for the Talon.
> 
> Adding that right now actually!

Pfft, nice timing, then.

**Review by Novacat — APPROVED** [review]

**Comment by VictorZisthus** [comment]
Todo: fix issue #8830 if this PR is still open tomorrow, else make a new PR

**Pull request merged**